extensions/resource-deployment/src/ui/deployAzureSQLDBWizard/pages/databaseSettingsPage.ts

```diff
@@ -139,9 +184,32 @@ export class DatabaseSettingsPage extends BasePage {
 		});
 	}
 
+	private validateDatabaseNameText(databasename: string | undefined): boolean {
```

Comment:
Can you rename the function? Since you are using the same function to validate both database name and collation.